game/transfers.py

```diff
@@ -571,6 +571,13 @@ def __init__(self, game: Game, player: bool) -> None:
     def __iter__(self) -> Iterator[TransferOrder]:
         yield from self.pending_transfers
 
+    def _update_eventstream(self) -> None:
+        from game.server import EventStream
+        from game.sim import GameUpdateEvents
+
+        events = GameUpdateEvents().update_supply_routes()
+        EventStream.put_nowait(events)
+
```

Comment:
```suggestion
    def _update_eventstream(self) -> None:
        from game.server import EventStream

        with EventStream.event_context() as events:
            events.update_supply_routes()
```

Slightly cleaner IMO...